assets/github_testing/test.yaml

```diff
@@ -1,8 +1,9 @@
 assembly_path: /home/runner/work/ascc/ascc/asccTinyTest/assembly/Pyoeliiyoelii17XNL_assembly.fa
 assembly_title: asccTinyTest
-pacbio_barcodes: /home/runner/work/ascc/ascc/assets/pacbio_adaptors.fa
+pacbio_barcodes: /home/runner/work/ascc/ascc/pacbio_barcode/pacbio_adaptors.fa
 pacbio_multiplexing_barcode_names: "bc1008,bc1009"
-pacbio_reads_path: /home/runner/work/ascc/ascc/asccTinyTest/pacbio/
+pacbio_reads_path: /home/runner/work/ascc/ascc/asccTinyTest/pacbio
+platform: "hifi"
```

Comment:
What's `platform` here is called `reads_type` in the `https://github.com/sanger-tol/cobiontcheck/blob/develop/asg_cobiont_check.py`, so for consistency it might be better to rename it here? But `platform` is okay too, I guess